src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/RouteConfig.kt

```diff
@@ -6,25 +6,38 @@ import org.vaccineimpact.orderlyweb.app_start.routing.web.*
 import org.vaccineimpact.orderlyweb.controllers.web.AdminController
 import org.vaccineimpact.orderlyweb.controllers.web.IndexController
 
+interface RouteBuilder
+{
+    fun getEndpoints(useAuth: Boolean): List<EndpointDefinition>
+}
+
 interface RouteConfig
 {
     val endpoints: List<EndpointDefinition>
 }
```

Comment:
Can `RouteConfig` interface be removed?

Reply:
no, it's still used for the most part. it's only the top level routes that use the new "builder" interface. have renamed those objects to make that clearer!